src/kibana/components/vislib/visualizations/pie_chart.js

```diff
@@ -58,8 +58,9 @@ define(function (require) {
      * @returns {D3.Selection} SVG with paths attached
      */
     PieChart.prototype.addPath = function (width, height, svg, slices) {
+      var margin = 10;
```

Comment:
Since we're setting the margin in code can we make the size of the margin a factor of the radius of the pie chart? Otherwise its easy to end up in a situation where the margin is a significant portion of the element:

![screen shot 2015-02-09 at 12 55 11 pm](https://cloud.githubusercontent.com/assets/1250387/6114380/fbaccc44-b05a-11e4-9ee7-f94cb6c93522.png)